src/App.js

```diff
@@ -109,86 +131,73 @@ function App() {
         <Paper elevation={0} square>
           <BrowserRouter>
             <div className={classes.root}>
+              <Sidebar
+                animation="bubble"
+                right
+                handleClick={handleClick}
+                handleClock={handleClose}
+                StyleTabs={StyledTabs}
+                StyleTab={StyledTab}
+                darkMode={darkMode}
+                setDarkMode={setDarkMode}
+                anchorEl={anchorEl}
+              />
               <Route
                 path="/"
                 render={({ location }) => (
                   <Fragment>
-                    {
-                      // This block is navbar at the top of the page
-                    }
-                    <nav
-                      class="navbar navbar-expand-lg navbar-light"
+                    <AppBar
+                      position={"fixed"}
                       style={{
                         boxShadow: "none",
-                        paddingTop: "10",
+                        paddingTop: 10,
```

Comment:
Not sure if this padding is necessary.
Also might be a good idea to give this appbar a set height (maybe 100px or something?) - will make it easier to add padding for other pages.

Reply:
remove the padding and add a height attribute